src/lib/utils/index_column_id_before_pruning.cpp

```diff
@@ -0,0 +1,21 @@
+#include "index_column_id_before_pruning.hpp"
+
+#include <vector>
+
+namespace opossum {
+
+ColumnID index_column_id_before_pruning(ColumnID index_column_id,
```

Comment:
We use this utility function to calculate the correct index column id. However, it can also be used to calculate an original column id, i.e., the corresponding column id before pruning, if the column id is not used in the context of indexes.

I would name it "column_id_before_pruning", analogous to "column_ids_after_pruning".

Reply:
Good point!